.github/workflows/test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
   # separate job for parallelism
   lint:
     name: Lint
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
why exactly is CI failing on 24.04?
I can't tell the reason from the failed PR as in not sure why the new ubuntu is causing failures..
Is there an upstream github issue somewhere - I assume this is not unique to our project?